packages/libp2p/test/connection-manager/auto-dial.spec.ts

```diff
@@ -22,7 +22,7 @@ import type { PeerStore, Peer } from '@libp2p/interface/peer-store'
 import type { ConnectionManager } from '@libp2p/interface-internal/connection-manager'
 
 describe('auto-dial', () => {
-  let autoDialler: AutoDial
+  let autoDial: AutoDial
```

Comment:
I think `autoDialler` is the correct language. Generally it's more appropriate to name components with nouns since as opposed to a verb.

Reply:
Ok, will replace with `autoDialer` to be consistent with "dialer" used elsewhere.